### PR TITLE
APP-160: [C] Basic level Apps: Configuration

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @forGGe @slash32

### CODE REVIEW

**Comment by Slash32** [comment]
test this please

**Comment by Slash32** [comment]
Looks good to me

File: configurationdemo/source/c/src/kaa_demo.c [hunk 1]
```diff
@@ -85,28 +75,41 @@ int main(/*int argc, char *argv[]*/) {
...
+            endpoint_key_hash);
+
     /**
      * Start Kaa client main loop.
      */
     error_code = kaa_client_start(kaa_client, NULL, NULL, 0);
     if(error_code) {
         demo_printf("Failed to start Kaa main loop\r\n");
-        return 4;
+        return 1;
```
Comment (forGGe) [inline configurationdemo/source/c/src/kaa_demo.c]: If you prefer return same values in all error cases, consider using `EXIT_SUCCESS` and `EXIT_FAILURE` values.

**Comment by forGGe** [comment]
Small nitpick. Looks good otherwise.

**Comment by vadimol** [comment]
+

**Pull request merged**